package/crossplane.yaml.tmpl

```diff
@@ -1,24 +1,28 @@
 apiVersion: meta.pkg.crossplane.io/v1alpha1
 kind: Provider
 metadata:
-  name: provider-gcp-{{ APIGROUP }}
+  name: {{ .Name }}
+{{ if ne .Service "monolith" }}
   labels:
-    pkg.crossplane.io/provider-family: gcp
+    pkg.crossplane.io/provider-family: provider-family-gcp
```

Comment:
nit: Should we flow the `CSP` makefile variable down to here?

Reply:
Done.